### PR TITLE
update docker image

### PR DESCRIPTION
**Pull request by talzich**
## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Content Pull Request
[Related PR: link to the PR at demisto/content](https://github.com/demisto/content/pull/33384)

## Related Issues
Related: link to the issue

## Description
Stringsifter is likely using scikit-learn v1.3.2 while our test are raising an error requesting v1.3.0. This PR aims to test whether the version change helps with the tests.

### CODE REVIEW

File: docker/stringsifter/pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+name = "dockerfiles"
+version = "0.1.0"
+description = ""
+authors = ["TalZich <tzichlinsky@paloaltonetworks.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "~3.10"
+stringsifter = "3.20230711"
+wheel = "*"
```
Comment (JudahSchwartz) [inline docker/stringsifter/pyproject.toml:11]: why do we need wheel?

File: docker/stringsifter/pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+scikit-learn = "1.3.0"
```
Comment (JudahSchwartz) [inline docker/stringsifter/pyproject.toml:12]: make sure this is properly ignored in the dependabot config file since we want it to stay locked here

Comment (talzich) [reply docker/stringsifter/pyproject.toml:12]: done

File: docker/stringsifter/pyproject.toml (hunk 1 above)
Comment (talzich) [reply docker/stringsifter/pyproject.toml:11]: when trying to `pip install stringsifter` on my local machine, a `ModuleNotFoundError` kept popping up saying `No module named wheel`. I figured it's a requirement for stringsifter. should it not be here?

Comment (JudahSchwartz) [reply docker/stringsifter/pyproject.toml:11]: lets remove from the poetry and see if it comes with the lock

**Review by JudahSchwartz — APPROVED** [review]

**Pull request merged**